tutoraspects/templates/aspects/apps/superset/pythonpath/create_assets.py

```diff
@@ -61,26 +62,46 @@ def main():
 def create_assets():
     """Create assets from a yaml file."""
     roles = {}
+
+    for root, dirs, files in os.walk(ASSETS_PATH):
+        for file in files:
+            if not file.endswith(".yaml"):
+                continue
+
+            path = os.path.join(root, file)
+            with open(path, "r") as file:
+                asset = yaml.safe_load(file)
+                if not asset:
+                    continue
+
+                # Process the asset directly
+                if FILE_NAME_ATTRIBUTE not in asset:
+                    raise Exception(f"Asset {asset} has no {FILE_NAME_ATTRIBUTE}")
+                file_name = asset.pop(FILE_NAME_ATTRIBUTE)
+
+                # Find the right folder to create the asset in
+                for asset_name, folder in ASSET_FOLDER_MAPPING.items():
+                    if asset_name in asset:
+                        write_asset_to_file(asset, asset_name, folder, file_name, roles)
+                        break
+
     with open(ASSETS_FILE_PATH, "r") as file:
         extra_assets = yaml.safe_load(file)
 
-        if not extra_assets:
-            print("No extra assets to create")
-            return
+        if extra_assets:
```

Comment:
Now that the extra_assets is empty by default, did this worked if empty?

Reply:
Yes, it work correctly with `extra_assets` empty